docs/contributing.md

```diff
@@ -42,7 +42,10 @@ Clone this repository `git clone git@github.com:codeready-toolchain/toolchain-e2
 
 This repository provides you multiple Makefile targets that you can use - it depends on which version of KubeSaw operators you want to install.
 
-**NOTE**: If the cluster is an OSD cluster, then set the variable `IS_OSD=true` when running any of the Makefile targets (for example: `make dev-deploy-latest IS_OSD=true`).
+NOTE:
```

Comment:
NOTE:
Set the `DISABLE_KUBE_CLIENT_TLS_VERIFY` environment variable to `true` when the TLS connection to the cluster cannot be verified. For example: 
```
make appstudio-dev-deploy-latest DISABLE_KUBE_CLIENT_TLS_VERIFY=true
```